sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_logical_subscriptions_history_v1/query.sql

```diff
@@ -0,0 +1,121 @@
+  ) AS id,
+  history.valid_from,
+  history.valid_to,
+  history.id AS provider_subscriptions_history_id,
+  STRUCT(
+    CONCAT(
+      'Stripe-',
+      history.subscription.id,
+      '-',
+      subscription_item.id,
+      '-',
+      FORMAT_TIMESTAMP('%FT%H:%M:%E6S', history.subscription.start_date)
+    ) AS id,
+    'Stripe' AS provider,
+    IF(paypal_subscriptions.subscription_id IS NOT NULL, 'PayPal', 'Stripe') AS payment_provider,
+    history.subscription.id AS provider_subscription_id,
+    subscription_item.id AS provider_subscription_item_id,
+    history.subscription.created AS provider_subscription_created_at,
+    history.subscription.customer_id AS provider_customer_id,
+    history.customer.metadata.userid AS mozilla_account_id,
+    history.customer.metadata.userid_sha256 AS mozilla_account_id_sha256,
+    COALESCE(
+      NULLIF(history.customer.shipping.address.country, ''),
+      NULLIF(history.customer.address.country, '')
+      -- TODO: also use charge billing addresses
+    ) AS country_code,
+    plan_services.services,
+    subscription_item.plan.product.id AS provider_product_id,
+    subscription_item.plan.product.name AS product_name,
+    subscription_item.plan.id AS provider_plan_id,
+    subscription_item.plan.`interval` AS plan_interval,
+    subscription_item.plan.interval_count AS plan_interval_count,
+    UPPER(subscription_item.plan.currency) AS plan_currency,
+    (CAST(subscription_item.plan.amount AS DECIMAL) / 100) AS plan_amount,
+    IF(ARRAY_LENGTH(plan_services.services) > 1, TRUE, FALSE) AS is_bundle,
+    IF(
+      history.subscription.status = 'trialing'
+      OR (
+        history.subscription.ended_at
+        BETWEEN history.subscription.trial_start
+        AND history.subscription.trial_end
+      ),
+      TRUE,
+      FALSE
+    ) AS is_trial,
+    -- The `is_active` logic should agree with what SubPlat considers an active Stripe subscription.
+    -- https://github.com/mozilla/fxa/blob/56026cd08e60525823c60c4f4116f705e79d6124/packages/fxa-shared/subscriptions/stripe.ts#L19-L24
+    history.subscription.status IN ('active', 'past_due', 'trialing') AS is_active,
+    history.subscription.status AS provider_status,
+    history.subscription.start_date AS started_at,
+    history.subscription.ended_at,
+    -- TODO: ended_reason
+    IF(
+      history.subscription.ended_at IS NULL,
+      history.subscription.current_period_start,
+      NULL
+    ) AS current_period_started_at,
+    IF(
+      history.subscription.ended_at IS NULL,
+      history.subscription.current_period_end,
+      NULL
+    ) AS current_period_ends_at,
+    history.subscription.cancel_at_period_end IS NOT TRUE AS auto_renew,
+    IF(
+      history.subscription.cancel_at_period_end,
+      history.subscription.canceled_at,
+      NULL
+    ) AS auto_renew_disabled_at
+    -- TODO: promotion_codes
+    -- TODO: promotion_discounts_amount
+    -- TODO: has_fraudulent_charges
```

Comment:
For expediency's sake I'm leaving out some of the less critical and/or more complicated fields/logic for now, and will add those in a later PR.